WordPress/Classes/Utility/Delay.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+
+typealias Action = () -> Void
+
+/// Something that can be canceled
+///
+protocol Cancelable {
+    func cancel()
```

Comment:
If `DispatchDelayedAction` is the one who implements this protocol, maybe we should consider... just implementing this method there?